test/integration/event_page_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'test_helper'
+
+feature 'Event' do
+  def setup
+    @user = make_user
+    @admin = make_admin
+  end
+
+  test 'shows a logo image if the event has a logo' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true, logo: 'logo-image')
+
+    visit events_path
+
+    assert page.text.include?('The Event')
+    assert page.body.include?('logo-image')
+    assert_not page.body.include?('event-default')
+  end
+
+  test 'shows a default image if the event does not have a logo' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true)
+
+    visit events_path
+
+    assert page.text.include?('The Event')
+    assert page.body.include?('event-default')
+  end
+
+  test 'shows Your events in the breadcrumb is the user is an organizer' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true, organizer_id: @user.id)
+
+    visit events_path
+
+    click_link 'The Event'
+
+    assert page.text.include?('Your Events')
+  end
+
+  test 'shows Events in the breadcrumb is the user is not an organizer' do
+    sign_in_as_user
+    make_event(name: 'The Event', approved: true)
+
+    visit events_path
+
+    click_link 'The Event'
+
+    assert page.text.include?('Events')
+  end
+
+  test 'shows Admin in the breadcrumb is the user is an Admin' do
+    sign_in_as_admin
+    make_event(name: 'The Event', approved: true, organizer_id: @user.id)
+
+    visit events_path
+
+    click_link 'The Event'
+
+    assert page.text.include?('Admin')
```

Comment:
this is not checking exactly what we want, as in the menu the word "Admin" is also appearing. maybe we should use an xpath to check more precisely the breadcrumb and then use it in all the tests related.